src/cloud-cli/meltano/cloud/api/client/__init__.py

```diff
@@ -228,8 +250,30 @@ async def run_project(
                 "meltano-runner-secret": self.runner_secret,
             }
         ):
-            return await self._request(
+            return await self._json_request(
                 "POST",
                 f"/{tenant_resource_key}/{project_id}/{deployment}/{job_or_schedule}",
                 base_url=self.runner_api_url,
             )
+
+    @asynccontextmanager
+    async def stream_logs(
+        self,
+        execution_id: str,
+    ) -> t.AsyncGenerator[ClientResponse, None]:
+        """Stream logs from a Meltano Cloud execution.
+
+        Args:
+            execution_id: The execution identifier.
+
+        Yields:
+            The response object.
+        """
+        url = (
+            "/logs/v1/"
+            f"{self.config.organization_id}/{self.config.project_id}/{execution_id}"
```

Comment:
@edgarrmondragon @cjohnhanson We should probably rename `organization_id` to `tenant_resource_key`, to avoid confusing these distinct concepts.

Reply:
Yeah, I agree. We're calling it different things in different contexts.